core_java/jdbc/sql/product.sql

```diff
@@ -0,0 +1,33 @@
+CREATE SEQUENCE hp_product_seq start with 10000;
```

Comment:
You do not have to create a sequence. You can just use `AUTO_INCREMENT` for integers.

Reply:
I attempted to implement AUTO_INCREMENT, but found that it is not supported with PostgreSQL. Instead I changed the product_id to be a SERIAL type to allow it to increment with every entry without the use of a sequence.